examples/with-route-as-modal-page/pages/index.js

```diff
@@ -0,0 +1,8 @@
+import * as PostPage from './post/[postId]'
+
+export const getStaticProps = () =>
+  PostPage.getStaticProps({
```

Comment:
This can be confusing to people, potentially you can do:

```
import {default as PostPage, getStaticProps as PostPageGetStaticProps} from './post/[postId]'
```